spec/rails_helper.rb

```diff
@@ -35,7 +35,12 @@
 
 # Checks for pending migration and applies them before tests are run.
 # If you are not using ActiveRecord, you can remove this line.
-ActiveRecord::Migration.maintain_test_schema!
+
+# As of Raild 6 upgrade, this causes an error:
+# PG::ConnectionBad:
+#  connection is closed
+# Likely due to some changed order of operations
+# ActiveRecord::Migration.maintain_test_schema!
```

Comment:
That error comes up in the app currently -- you just have to re-run the command. I'm not sure why it's happening and agree that it probably shouldnt, but this isn't a Rails 6 specific issue.

Reply:
I'm getting it consistently on my machine, and I don't get it on master. Maybe it's a bit of both, sprinkled with a little OS X funtimes?

Reply:
ha -- That's possible. 

This issue looks related: https://github.com/rails/rails/issues/37407

It might be an upstream problem. What's the consequences of leaving this commented for now? We just have to do `RAILS_ENV=test bundle exec rails db:migrate` after running regular migrations?